app/models/ranking/popular_posts.rb

```diff
@@ -0,0 +1,54 @@
+class Ranking::PopularPosts
```

Comment:
Hmm, we may have better name for this class.
Could you describe the responsibility of this class?